Dockerfile

```diff
@@ -1,23 +1,28 @@
 FROM ubuntu:14.04
```

Comment:
14.04 is EOL, so this should probably be updated as well. I'm assuming CDH now supports 16.04.

Reply:
Agreed.